src/SimpleApi/DependencyRegistrationExtension.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace SimpleApi
+{
+    public static class DependencyRegistrationExtension
+    {
+        public static IApplicationBuilder RunStartupTasks(this IApplicationBuilder app)
+        {
+            if (app == null) throw new ArgumentNullException(nameof(app));
+
+            using (var serviceScope = app.ApplicationServices.CreateScope())
+            {
+                var startupTasks = serviceScope.ServiceProvider.GetServices<IStartupTask>();
+
+                foreach (var startupTask in startupTasks)
+                {
+                    startupTask.ExecuteAsync();
+                }
+            }
+
+            return app;
+        }
+    }
```

Comment:
Should this be added to the library rather than the example?

Reply:
We already have the MigrationRunner.Migrate to run the migration. I think it should be user's responsibility to run the migration as per the need. This above code is running all startup tasks which I don't think should be the responsibility of Mongration library.  Hence I added this in sample API